agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -197,6 +197,15 @@ function collect_insights_data {
 	fi
 }
 
+function collect_PSI_data {
+	debug_log "[$script_name]collecting system PSI data"
+        if [ -d  /proc/pressure ] ;then
+                cat /proc/pressure/memory >> $dir/PSI_memory.txt
+                cat /proc/pressure/io >> $dir/PSI_io.txt
+                cat /proc/pressure/cpu >> $dir/PSI_cpu.txt
```

Comment:
Can you consider writing this as:

```
mkdir ${dir}/PSI
if [[ ${?} -ne 0 ]]; then
        error_log "[${script_name}] failed to create PSI collection directory, ${dir}/PSI"
        exit 1
fi
for i in cpu io memory; do
        cat /proc/pressure/${i} > ${dir}/PSI/${i}.txt
done
```

And notice we use the single greater-than-sign redirect to avoid appending to an existing file.